README.rst

```diff
@@ -20,33 +20,238 @@
+
+I think I found a bug. What should I do?
+----------------------------------------
+
+Please make an Issue describing the bug so we can address it, or work with you
+to address it. Please try to provide a
+`minimal, reproducible example <https://stackoverflow.com/help/minimal-reproducible-example>`_.
+
+I found something in the documentation that isn't clear. What should I do?
+--------------------------------------------------------------------------
+
+Please make an Issue describing the what isn't clear to you. Someone will tag
+the most appropriate member of the core Landlab team. We will work to clarify
+your question and revise the documentation so that it is clear for the next user.
+
+I'm interested in contributing to Landlab. Where do I get started?
+------------------------------------------------------------------
+
+Thank you for your interest! Refer to ``CONTRIBUTING.md`` and
+`this <https://landlab.readthedocs.io/en/master/development/index.html#development>`_
+page in the documentation that describes contribution guidelines.
+
+How is the Landlab package structured?
+--------------------------------------
+
+The `following page <>` in the documentation describes the package structure. 
+
+How was Landlab funded?
+-----------------------
+
+Landlab is funded by the US National Science Foundation. It has been supported
+by the following grants:
+
+    * A Collaborative NSF SI2-SSE proposal to
+      University of Colorado (Greg Tucker,
+      `1147454 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1147454&HistoricalAwards=false>`_),
+      and the University of Washington (Erkan Istanbulluoglu,
+      `1148305 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1148305&HistoricalAwards=false>`_)
+    * A Collaborative NSF SI2-SSI proposal to
+      University of Colorado (Greg Tucker and Dan Hobley,
+      `1450409 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1450409&HistoricalAwards=false>`_),
+      Tulane University (Nicole Gasparini,
+      `1450338 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1450338&HistoricalAwards=false>`_),
+      and the University of Washington (Erkan Istanbulluoglu,
+      `1450412 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1450412&HistoricalAwards=false>`_).
+    * A NSF EAR Postdoctoral Fellowship to Katy Barnhart
+      (`1725774 <https://www.nsf.gov/awardsearch/showAward?AWD_ID=1725774&HistoricalAwards=false>`_).
+
+Who made Landlab?
+-----------------
+
+The core development team is currently composed of:
 
-The most current source code is always available from our git
-repository:
+ - Greg Tucker (CU)
+ - Nicole Gasparini (Tulane)
+ - Erkan Istanbulluoglu (UW)
+ - Daniel Hobley (Cardiff)
+ - Sai S. Nudurupati (UW)
+ - Jordan Adams (Tulane)
+ - Eric Hutton (CU)
+ - Jenny Knuth (CU)
+ - Katy Barnhart (CU)
+ - Margaux Mouchene (CU)
+ - Christina Bandaragoda (UW)
+ - Nathan Lyons (Tulane)
 
-https://github.com/landlab/landlab
+We welcome all community contributions.
```

Comment:
We welcome all community contributions. Please see https://github.com/landlab/landlab/blob/master/CONTRIBUTING.md.

Reply:
Just saw this was mentioned above. Probably not needed again here.

Reply:
good point. I've deleted it.